vunit/ui.py

```diff
@@ -191,6 +191,16 @@
    Extra simulation flags passed to ``ghdl --elab-run``.
    Must be a list of strings.
 
+``ghdl.gtkwave_script.gui``
+   A user defined TCL-file that is sourced after the design has been loaded in the GUI.
+   For example this can be used to configure the waveform viewer.
+   During script evaluation the ``vunit_tb_path`` variable is defined

```

Comment:
I do not see how vunit_tb_path tcl variable is available so this documentation is inaccurate. Maybe copy paste error?

Reply:
Yep, sorry about that
```suggestion
```